index.html

```diff
@@ -0,0 +1,193 @@
+                  <p class="card-text">All the guests will register for attendance</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/food-items.webp" class="card-img-top bg-white" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Collection of food items</h5>
+                  <p class="card-text">All food items will be collected and stored awaiting distribution.</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/sleep_pillow_blanket-256.webp" class="bg-white card-img-top" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Collection of non-food items</h5>
+                  <p class="card-text">All non-food items will be collected and stored awaiting distribution.</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/money.webp" class="card-img-top bg-white" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Cash fundraising</h5>
+                  <p class="card-text">Cash, money transfer and cheque will be collected.</p>
+                </div>
+              </div>
+            </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!--Featured Speakers Section -->
+
+      <section id="speaker-wrapper">
+        <h2 class="section-header text-center" id="speaker">Featured Speakers</h2>
+        <div class="guide_bar"></div>
+        <div class="container">
+            <div class="row p-2" id="featured-speakers">
+            </div>
+        </div>
+    </section>
+
+       
+
+      <section class="main-footer">
+        <h4 class="text-light mt-5 text-center" id="Partners">Main Partners</h4>
+        <div class="footer-sec">
+          <ul class="footers">
+            <li><img src="./image/kenya-flag.webp" alt="kenya Government"></li>
+            <li><img src="./image/un-logo.webp" alt="U.N"></li>
+            <li><img src="./image/microsoft-vector-logo.png" alt="Microsoft"></li>
+            <li><img src="./image/safaricom-logo.png" alt="Safaricom"></li>
+            <li><img src="./image/equity-bank-kenya-logo-hd-png-download.png" alt="Equity bank"></li>            
+            <li><img src="./image/kenya-airways.png" alt="Kenya Airways"></li>
+
+          </ul>
+        </div>
+      </section>
+      <footer>
+        <div class="footer_logo">
+            <img src="./image/logo-krcs.png" height="60" alt="kenya redcross logo">
+            <p>&copy; All rights Reserved</p>
+        </div>
+      </footer>
```

Comment:
Kindly modify the footer to match the design. According to the design, the footer note takes up about 3/4 of the available width.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192086127-fc00050a-a9ef-4d49-a482-dd2f2db90cd0.png)


Design guideline:
![image](https://user-images.githubusercontent.com/20554583/192086146-150becd8-cafe-42a6-874e-00b39f61c751.png)